packer/scripts/packager_freebsd.sh

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env sh
+set -e
+
+# Install development tools because we always need those
+pkg update
+pkg install -y curl
+pkg install -y gcc
+pkg install -y bash
+
+# Ensure that /dev/fd is mounted (required by bash)
+grep '/dev/fd\>' /etc/fstab > /dev/null || \
+  echo 'fdesc   /dev/fd         fdescfs         rw      0       0' >> /etc/fstab
+mount | grep '/dev/fd\>' > /dev/null || \
+  mount /dev/fd
+
+# Install it system-wide
+curl -sSL https://get.rvm.io | bash -s stable
+
+# Let this user use it and install Rubies
+pw groupmod rvm -M `whoami`
```

Comment:
This is acquired from ``packager_ubuntu.sh`` but if the script is run by ``sudo packer/scripts/packager_freebsd.sh`` it adds ``root`` to rvm group (at least when running manually). Not sure if it will work as intended with ``packer``. Comments?